doc/development/plugins.rst

```diff
@@ -269,6 +271,82 @@ this may have unintended side-effects and is not recommended.
 .. _installing_plugin:
 
 
+Wire handling
+-------------
+
+PennyLane uses the :class:`~.wires.Wires` class for the internal representation of wires. :class:`~.wires.Wires`
+inherits from python's ``Sequence``, and represents an ordered set of unique wire labels.
+Indexing a ``Wires`` instance will return another ``Wires`` instance of length one.
+The ``labels`` attribute stores a tuple of the wire labels.
+
+For example:
+
+.. code-block:: python
+
+    from pennylane.wires import Wires
+
+    wires = Wires(['ancilla', 0, 1])
+    print(wires[0]) # <Wires = ['ancilla']>
+    print(wires.labels) # ('ancilla', 0, 1)
+
+As shown in the section on :doc:`/introduction/circuits`, a device can be created with custom wire labels:
+
+.. code-block:: python
+
+    from pennylane import *
+
+    dev = device('my.device', wires=['q11', 'q12', 'q21', 'q22'])
+
+    @qnode(dev)
+    def circuit():
+       MyGate1(wires='q22')
+       MyGate2(wires=['q21','q11'])
+       MyGate1(wires=['q21'])
+       return expval(MyObs(wires='q11')@ MyObs(wires='q12'))
```

Comment:
Or shall I use "real" qubit gates here?

Reply:
I think this looks fine, although it seems like most other parts of the documentation use "real" gates in the examples. :thinking: